.github/workflows/tests.yml

```diff
@@ -118,3 +118,41 @@ jobs:
         # don't check /etc/os-release sourcing, allow useless cats to live inside our codebase, and
         # allow seemingly unreachable commands
         SHELLCHECK_OPTS: -e SC1091 -e SC2002 -e SC2317
+
+  python-test:
+    name: "🐍 pytest (imgtestlib)"
+    runs-on: ubuntu-latest
+    container:
+      image: registry.fedoraproject.org/fedora:latest
+    steps:
+
+      - name: Install build and test dependencies
+        run: dnf -y install python3-pytest
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Testing imgtestlib
+        run: |
+          python3 -m pylint -v
```

Comment:
Should this be `python3 -m pytest` instead of *pylint* ? (or just `pytest -v`)?

Reply:
😳